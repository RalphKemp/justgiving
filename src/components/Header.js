import React from "react";
import styled from "styled-components";
const image =
  "https://www.jg-cdn.com/chrome/3e7a4dc9d866fda15331414a9072da97.svg";

const HeaderContainer = styled.div`
  width: 100%;
  height: 50px;
  text-align: center;
  display: flex;
  flex-direction: column;
  align-items: flex-start;
  justify-content: center;
  margin-bottom: 30px;
  padding-left: 70px;
  img {
    width: 245px;
  }
`;

const Header = () => {
  return (
    <HeaderContainer>
      <img src={image} />
      <p>Charity Info</p>
    </HeaderContainer>
  );
};

export default Header;
