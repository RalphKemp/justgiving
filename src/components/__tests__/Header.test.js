import React from 'react';
import { shallow } from 'enzyme';
import Header from '../Header.js';

describe('Header', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
