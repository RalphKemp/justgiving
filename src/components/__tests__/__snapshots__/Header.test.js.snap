// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<styled.div>
  <img
    alt="JGlogo"
    src="https://www.jg-cdn.com/chrome/3e7a4dc9d866fda15331414a9072da97.svg"
  />
</styled.div>
`;
